### ISSUE TITLE
Some codecs not playing through Chromecast

### ISSUE THREAD

**Issue by markdancestango** [Taiko2k/Tauon#1269]
The built-in Chromecast function is very useful and good! thank you.

But it seems like some codecs (e.g. M4a/Aiff) are not able to be casted even though Tauon music box can play that through the local computer speaker.

Is this a bug or is it a restriction because of the Chromecast plug-in(?) setting?

What codecs can be natively cast to Chromecast?

Thanks.

**Comment by Taiko2k** [comment]
Yeah it's a known issue. The codecs that Chromecast accepts are limited.
The solution would be for Tauon to transcode any incompatible codecs on the
fly, but I just haven't allocated development time toward this feature, I'm
not sure this feature is used enough to make it a priority. I'd accept a PR
for it.
